prepare-release.sh

```diff
@@ -30,7 +30,7 @@ else
     ./update-versions.sh "${NEW_VERSION}" || exit $?
 
     # sanity check the diffs, but not Cargo.lock files cuz ugh
-    git diff -- . ':(exclude)Cargo.lock'
+    # git diff -- . ':(exclude)Cargo.lock'
```

Comment:
I have absolutely no idea why, but doing this `git diff` command would move me back to the `develop` branch causing the `git commit` below to go there.

Computers were a mistake.  I've never seen the script behave this way before.  I could be going crazy, but this happened 3 times.